pkgs/applications/networking/instant-messengers/discord/linux.nix

```diff
@@ -101,10 +101,12 @@ stdenv.mkDerivation rec {
     runHook postInstall
   '';
 
-  postInstall = lib.strings.optionalString (openasar != null) ''
+  postInstall = lib.strings.optionalString withOpenASAR ''
     cp -f ${openasar} $out/opt/${binaryName}/resources/app.asar
   '';
 
+  buildInputs = lib.optionals withOpenASAR [ unzip ];
```

Comment:
That is most likely nativeBuildInputs

Reply:
I checked without this line, and unzip is being added to "the closure" of the discord package, which is the right place for it (it's needed in runtime).